src/test/ui/const-generics/unknown_adt.rs

```diff
@@ -1,5 +1,10 @@
-#![feature(const_generics)]
-#![allow(incomplete_features)]
+// Checks errors when there is an abstract data type.
```

Comment:
```suggestion
```
I feel like this line doesn't help much/is slightly incorrect rn.

This test checks that `type_of` does not break for const args on unknown types.

Also, adt means "algebraic data type", not "abstract dt" :sparkles:

Reply:
ah I'll just remove it then